draft-pauly-dprive-oblivious-doh.md

```diff
@@ -191,7 +198,7 @@ Targets MUST return 4xx (Client Error) if this check fails.
 ## HTTP Request Example {#request-example}
 
 The following example shows how a Client requests that a Proxy, "dnsproxy.example",
-forwards an encrypted message to "dnstarget.example". The URI template for the
+forwards an encrypted message to "dnstarget.example". The URI Template for the
```

Comment:
This example mentions two URI templates: one for the proxy and one for the target. The normative text above should describe both (it only describes the first one right now). But if I'm reading this document correctly, the "Target URI Template" is not a URI template, it's just a URI. So I'd say in the normative section that the Client is configured with both a Proxy URI Template and a Target URI. (Oh and while we're at it, say that the scheme for both MUST be HTTPS and clients MUST check that)

Reply:
I _think_ a template without any parameters is still a valid template, but you're right that this is probably confusing. Took the suggestion and fixed. 👍